covid_recipes/atomicity_in_apply.yml

```diff
@@ -0,0 +1,190 @@
+            {{ start_key + 10 }},AUS,15-03-2021,120000,95000,25000,3000,5000,4.8,3.7,1.1,1520,
+            {{ start_key + 11 }},AUT,20-03-2021,87500,64000,23500,2200,4300,5.2,3.8,1.4,980,
+            {{ start_key + 12 }},AZE,28-03-2021,60500,48200,12300,1800,2900,3.4,2.7,0.7,1100,
+            {{ start_key + 13 }},BHS,02-04-2021,32000,24500,7500,900,1500,8.1,6.2,1.9,890,
+            {{ start_key + 14 }},BHR,05-04-2021,76000,58000,18000,2600,3700,12.3,9.4,2.9,1440,
+            {{ start_key + 15 }},BGD,09-04-2021,210000,166000,44000,7200,9500,1.7,1.3,0.4,1270,
+            {{ start_key + 16 }},BRB,12-04-2021,28750,20900,7850,1100,1500,9.8,7.1,2.7,1340,
+            {{ start_key + 17 }},BLR,18-04-2021,97000,71400,25600,3400,4900,10.1,7.4,2.7,1420,
+            {{ start_key + 18 }},BEL,25-04-2021,134000,100200,33800,4500,6900,11.5,8.6,2.9,1590,
+            {{ start_key + 19 }},BLZ,02-05-2021,24300,17600,6700,800,1300,6.4,4.6,1.8,930,
+            {{ start_key + 20 }},BOL,08-05-2021,86500,65000,21500,2600,4100,5.7,4.3,1.4,1050,
+            {{ start_key + 21 }},BRA,15-05-2021,452000,355000,97000,13500,17500,6.2,4.9,1.3,2100,
+            {{ start_key + 22 }},BRN,20-05-2021,19400,14200,5200,700,1200,7.1,5.2,1.9,870,
+            {{ start_key + 23 }},BGR,28-05-2021,65500,49800,15700,1900,3200,5.5,4.2,1.3,1150,
+            {{ start_key + 24 }},BFA,03-06-2021,30700,22100,8600,1000,1600,3.8,2.9,0.9,940,
+            {{ start_key + 25 }},BDI,10-06-2021,22900,16800,6100,800,1300,4.2,3.2,1.0,910,
+            {{ start_key + 26 }},KHM,17-06-2021,74300,56200,18100,2400,3700,7.9,5.9,2.0,1280,
+            {{ start_key + 27 }},CMR,24-06-2021,98400,75600,22800,3200,4800,8.3,6.4,1.9,1330,
+            {{ start_key + 28 }},CAN,01-07-2021,152000,115000,37000,5200,7200,9.6,7.2,2.4,1750,
+            {{ start_key + 29 }},CPV,08-07-2021,18400,13400,5000,600,900,6.7,4.9,1.8,880,
+          dest: "{{ NIOVA_BIN_PATH }}/{{ csv_file2 }}"
+
+      - name: "{{ recipe_name }}: Write monotonic keys to COVID Application."
+        vars:
+          input_param: {
+                async_mode: True     # run in async mode
+          }
+          stage: "CovidWriteMulti2"
+          cmd: "WriteMulti#{{ NIOVA_BIN_PATH }}/{{ csv_file2 }}#{{ recipe_name }}_{{ stage }}"
+          result: "{{ lookup('app_cmd', cmd, client_uuid.stdout, input_param) }}"
+        debug:
+          msg: "Launched async multiwrite: {{ result }}"
+
+      - pause:
+          seconds: 2
+
+      ####################################################################
+      # 6. KILL THE LEADER
+      ####################################################################
+      
+      - name: "{{ recipe_name }}: Kill the leader while writes are in progress." 
+        debug: 
+          msg: "{{ lookup('niova_raftprocess', 'kill', current_leader_uuid) }}" 
+        no_log: false
+
+      ####################################################################
+      # 7. VERIFY PERSISTENCE
```

Comment:
I see that `get_last_written_key` is implemented in verify_rocksdb.py, But where is it being used? 

Rename comment from VERIFY PERSISTENCE -> VERIFY AUTOMICITY and integrate the final steps.